db/vita_partners.yml

```diff
@@ -51,71 +78,95 @@ vita_partners:
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009397734"
     display_name: Foundation Communities
-    source_parameter: fc
+    source_parameters:
+      - fc
+    states:
+      - TX
     logo_path: ""
   - name: United Way of Central Ohio
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009440374"
     display_name: United Way of Central Ohio
-    source_parameter: uwco
+    source_parameters:
+      - uwco
+    states:
+      - OH
     logo_path: ""
   - name: Nevada Free Taxes Coalition
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009537374"
     display_name: Nevada Free Taxes Coalition
-    source_parameter: nvftc
+    source_parameters:
+      - nvftc
+    states:
+      - NV
     logo_path: ""
   - name: United Way of Tuscon and Southern Arizona
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009581934"
     display_name: United Way of Tuscon and Southern Arizona
-    source_parameter:
     logo_path: ""
+    accepts_overflow: true
   - name: United Way of Greater Richmond and Petersburg
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009267673"
     display_name: United Way of Greater Richmond and Petersburg
-    source_parameter: uwvp
-    logo_path: "partner-logos/uwvp.jpg"
+    source_parameters:
+      - uwvp
+    states:
+      - VA
+    logo_path: ""
   - name: United Way Fresno Madera
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009708233"
     display_name: United Way Fresno Madera
-    source_parameter: uwfm
-    logo_path: "partner-logos/uwfm.png"
+    source_parameters:
+      - uwfm
+    logo_path: ""
   - name: United Way California Capital Region
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009708193"
     display_name: United Way California Capital Region
-    source_parameter: uwccr
-    logo_path: "partner-logos/uwccr.jpg"
+    source_parameters:
+      - uwccr
+    logo_path: ""
   - name: Branches
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009704234"
     display_name: Branches
-    source_parameter: branchesfl
+    source_parameters:
+      - RefundDay-B
+      - branchesfl
     logo_path: ""
   - name: Hispanic Unity
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009704314"
     display_name: Hispanic Unity
-    source_parameter: hispanicunity
+    source_parameters:
+      - RefundDay-H
+      - hispanicunity
     logo_path: ""
   - name: Catalyst Miami
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009704354"
     display_name: Catalyst Miami
-    source_parameter: catalyst
+    source_parameters:
+      - catalyst
+      - RefundDay-C
     logo_path: ""
   - name: RefundDay
     zendesk_instance_domain: eitc
     zendesk_group_id: "360009657014"
     display_name: RefundDay
-    source_parameter:
-    logo_path: "partner-logos/refund-day.png"
+    source_parameters:
+      - RefundDay
```

Comment:
Yeah, this is a prefix of the other RefundDay sources, so I would want to have a bit more confidence that the routing logic properly works for the other ones if you're going to add this one as part of this branch.